code/modules/mob/living/silicon/pai/software/pai_apps.dm

```diff
@@ -317,16 +317,17 @@
 					to_chat(usr, "<span class='warning'>You are already hacking that door!</span>")
 				else
 					hacking = TRUE
-					INVOKE_ASYNC(src, /datum/pai_software/door_jack/.proc/hackloop)
+					INVOKE_ASYNC(src, .proc/hackloop)
 		if("cancel")
 			hackdoor = null
 		if("cable")
-			if(cable)
-				to_chat(usr, "<span class='warning'>You already have a cable deployed!</span>")
-				return
-			var/turf/T = get_turf(pai_holder)
-			cable = new /obj/item/pai_cable(T)
-			pai_holder.visible_message("<span class='warning'>A port on [pai_holder] opens to reveal [cable], which promptly falls to the floor.</span>")
+			playsound(pai_holder, 'sound/mecha/mechmove03.ogg', 25, TRUE)
```

Comment:
This is such a pathetically weak way of hacking, I don't see the need to nerf it.

Reply:
The fastest the door can be hacked is 5 seconds, which is just 2 seconds over an access tuner. While it's very unlikely to actually be this fast I'd still consider it in need of *some* sort of downside.